src/components/Book.js

```diff
@@ -4,12 +4,12 @@ import { removeBookAction } from '../redux/books/books';
 
 const Book = (props) => {
   const { book } = props;
-  const { id, title, author } = book;
+  const { item_id: itemId, title, author } = book;
 
   const dispatch = useDispatch();
 
   const removeBook = () => {
-    dispatch(removeBookAction(id));
+    dispatch(removeBookAction(itemId));
   };
 
```

Comment:
### Before Fix
![image](https://user-images.githubusercontent.com/93039359/214788221-ff2f91f0-fe1d-48d5-8c5e-6a1ef1bde416.png)

### After Fix
![image](https://user-images.githubusercontent.com/93039359/214788118-e9098dfb-37c3-4fb4-abfe-d899e77f4753.png)

- [ ] You are disabling linters which is not good for your project because linters prevent you to make those errors in your project which can break your application :fearful: I have given you the solution in the second image as well :+1: